docs/src/components/Demo.tsx

```diff
@@ -0,0 +1,53 @@
+import { useState } from 'react';
+import { CopyToClipboard } from 'react-copy-to-clipboard';
+import Highlight, { defaultProps } from 'prism-react-renderer';
+import { Tabs, TabItem, Card, Flex, View, Button } from '@aws-amplify/ui-react';
+
+export const Demo = ({ children, propControls, themeControls, code }) => {
```

Comment:
This is the component that will be used for primitive demos. It handles the layout and code highlighting and copy button.

Reply:
💯